snyk-import/README.md

```diff
@@ -0,0 +1,36 @@
+# Snyk Import Extension
+
+This extension imports Snyk projects into Phylum by retrieving dependency
+information from the Snyk API and submitting the dependencies as a new analysis
+job to a Phylum project of the same name.
+
+## Installation
+
+Clone the repository and install the extension via the Phylum CLI.
+
+```console
+git clone https://github.com/phylum-dev/community-extensions
+phylum extension install community-extensions/snyk-import/
+```
+
+## Running
+
+To import all of your current Snyk projects
+
+```
+phylum snyk-import --group <phylum_group>
+```
+
+Or, setup a [Snyk service account][] and use the token to import all projects in
```

Comment:
```suggestion
Or, setup a [Snyk service account] and use the token to import all projects in
```

This should work just fine if you don't want to give it a separate ID.

Reply:
They both work... I've just always preferred this style (called [collapsed reference links][1]). But I believe I'm outnumbered on this point...

[1]: https://spec.commonmark.org/0.30/#collapsed-reference-link